luma/core/interface/serial.py

```diff
@@ -490,3 +490,292 @@ def ftdi_i2c(device='ftdi://::/1', address=0x3C):
     serial._managed = True
     serial._i2c_msg_write = lambda address, data: (address, data)
     return serial
+
+
+@lib.rpi_gpio
+class parallel(object):
+    """
+    Implements a parallel bus style interface for HD44780 style displays that
+    provides :py:func:`data` and :py:func:`command` methods. The default pin
+    assignments provided are from `Adafruit <https://learn.adafruit.com/drive-a-16x2-lcd-directly-with-a-raspberry-pi/wiring>`.
+
+    :param gpio: GPIO interface (must be compatible with
+        `RPi.GPIO <https://pypi.python.org/pypi/RPi.GPIO>`_)
+    :param pulse_time: length of time in seconds that the enable line should be
+        held high during a data or command transfer
+    :type pulse_time: float
+    :param RS: The GPIO pin register select (RS) line (low for command, high
+        for data)
+    :type RS: int
+    :param E: The GPIO pin to connect the enable (E) line to.
+    :type E: int
+    :param PINS: The GPIO pins that form the data bus (a list of 4 or 8 pins
+        depending upon implementation ordered from LSD to MSD)
+    :type PINS: list[int]
+
+    .. versionadded:: 1.15.0
+    """
+
+    from time import sleep
```

Comment:
this import should be moved to top of the module.

Reply:
Ok.  There's some other instances (in luma.lcd) where I've imported inside classes to avoid having the import show up in the modules namespace.  I'll change those as well before submitting.